apps/datahub/src/index.html

```diff
@@ -5,6 +5,9 @@
     <title>Datahub</title>
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta property="og:title" content="Datahub" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="" />
```

Comment:
Is this really required or will the angular code be enough to create those elements?